changelog/435.deprecate.rst

```diff
@@ -0,0 +1 @@
+``EmptyEmbed`` and ``Embed.Empty`` are deprecated in favor of ``None``.
```

Comment:
and have been removed from the documentation

Reply:
@shiftinv you missed this file too :^)

Reply:
um, no I definitely didn't :^) https://github.com/DisnakeDev/disnake/pull/768/commits/7b7b0be0ef6ea414444cad4875597ef9b9e596b6